teachopencadd/talktorials/T014_binding_site_detection/talktorial.ipynb

```diff
@@ -0,0 +1,2116 @@
+{
```

Comment:
<p>Will look into this after the review.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>Thanks!</p>

Reply:
When I work with Jupyter Notebook, your code shows the volume - in JLab it does not. I added a comment to the code.

Could not make it work from the ccp4 URL directly, so I kept your downloading step and added a `filepath.unlink()` after visualization to remove the file again.

Reply:
Well happy that it works! Weird that it does not show in JLab. Thanks for adding the respective comment.